crates/bevy_render/src/entity.rs

```diff
@@ -77,3 +86,33 @@ impl Default for Camera2dBundle {
         }
     }
 }
+
+/// A component bundle for "orthographic camera" entities
+///
+/// Uses a normalized orthographic projection.
+///
+/// Use this for CAD-like 3D views, isometric games,
+/// or 2D games that should scale with the window.
+#[derive(Bundle)]
+pub struct CameraOrthoBundle {
+    pub camera: Camera,
+    pub orthographic_projection: ScaledOrthographicProjection,
+    pub visible_entities: VisibleEntities,
+    pub transform: Transform,
+    pub global_transform: GlobalTransform,
+}
+
+impl Default for CameraOrthoBundle {
+    fn default() -> Self {
+        CameraOrthoBundle {
+            camera: Camera {
+                name: Some(base::camera::CAMERA_3D.to_string()),
```

Comment:
Should the new camera use the `CAMERA_3D` or `CAMERA_2D` name? What purpose do these serve / what difference would it make?